techniques/fileDistribution/fileTemplate/1.0/metadata.xml

```diff
@@ -0,0 +1,148 @@
+<TECHNIQUE name="File content (from remote template)">
+
+  <DESCRIPTION>This technique load a mustache/jinja template from the server's shared-files or a text input and expand it on the given location. Variables needed to expand the templates must already be loaded.
+    For jinja2 template type python must also be installed on the agent.
+    A limit of 16.000 characters is set for the text input template version.
+  </DESCRIPTION>
+  <MULTIINSTANCE>true</MULTIINSTANCE>
+
+
+  <TMLS>
+    <TML name="fileTemplate"/>
+  </TMLS>
+
+  <BUNDLES>
+    <NAME>fileTemplate</NAME>
+  </BUNDLES>
+
+  <TRACKINGVARIABLE>
+    <SAMESIZEAS>FILE_TEMPLATE_AGENT_DESTINATION_PATH</SAMESIZEAS>
+  </TRACKINGVARIABLE>
+
+  <SECTIONS>
+    <SECTION name="Apply template" multivalued="true" component="false">
+      <SECTION name="Load Template from a file or text input" multivalued="false" component="true" componentKey="FILE_TEMPLATE_AGENT_DESTINATION_PATH">
+        <SELECT1>
+          <NAME>FILE_TEMPLATE_RAW_OR_NOT</NAME>
+          <DESCRIPTION>Template source</DESCRIPTION>
+          <ITEM>
+            <VALUE>Raw</VALUE>
+            <LABEL>From Text</LABEL>
+          </ITEM>
+          <ITEM>
+            <VALUE>File</VALUE>
+            <LABEL>From a File</LABEL>
+          </ITEM>
+
+          <CONSTRAINT>
+            <DEFAULT>File</DEFAULT>
+          </CONSTRAINT>
+        </SELECT1>
+
+        <INPUT>
+          <NAME>FILE_TEMPLATE_TEMPLATE</NAME>
+          <DESCRIPTION>Name of the template file</DESCRIPTION>
```

Comment:
I think in the text case, you should create the name, since it's temporary and the user doesn't need to invent one or even to know it.
Example: "template_<directive_id>.tpl"